### ISSUE TITLE
Ordering at JLCPCB

### ISSUE THREAD

**Issue by Sineos** [zapta/ble_stepper_motor_analyzer#1]
Since I have never ordered something at JLC, some stupid questions:
1. 1.4.2 is good to go or are any major updated planned in the new future?
2. Ordering process:
   1. Upload `gerber.zip` --> Use JLC defaults
   2. Upload `bom.csv` and `positions.csv`
   3. C15,C18,R17,R18 are marked as `DNP` (= Do not populate / place?) --> Do not select them as well as the JTAG?
3. Anything else to mind?

Thanks for providing this cool design.

**Comment by zapta** [comment]
Hi Sineos,

1. I ordered and tested successfully 1.4.1 (and have three units in my
printer).  1.4.2, is the same as 1.4.1 but automatically migrated to Kicad
7. I didn't test 1.4.2  it myself but somebody else made a few tweaks to
that design and ordered successfully. The tweaks involve changing the
connectors to more common ones with 3D controllers from China.

If all you want is hassle free one or two boards, they may be available for
sale in a month or two. I am not involved but will link on the main page
anybody that make them available to the community.

2. That's correct. If you consider making firmware development yourself,
add also the JTAG connector, at least on the board you want to use for
development (it's not a must but having a hardware debugger is handy). If
you just want to make and use the standard firmware than no need for the
JTAG connectors.

BTW, after I ordered my last batch, I think that JLCPCB now requires
'standard' level service to assemble the ESP32 modules.

3. What else? You will also need antennas, connectors, etc. If you are
handy with soldering, when you test the first unit, remove r16, and verify
that you have 3.3v on c11 with usb and/or external power. r16 is a short
that was placed for this reason, to disconnect the circuitry from the power
supply.

Wish you success, and I will try to help if needed, but basically this
reference design is 'as is' with no warranties.

Zapta
